backend/src/main/kotlin/blue/mild/covid/vaxx/service/PatientService.kt

```diff
@@ -95,7 +95,7 @@ class PatientService(
             id = patientId,
             firstName = changeSet.firstName?.trim(),
             lastName = changeSet.lastName?.trim(),
-            phoneNumber = changeSet.phoneNumber?.trim(),
+            phoneNumber = if (changeSet.phoneNumber != null) validationService.formatPhoneNumber(changeSet.phoneNumber) else null,
```

Comment:
nelibi se mi, ze `validationService` vola `format` - as tohle formatovani by bylo mozna dobre nekam vyhodit (stejne tak ty trimy)

a tedy tady kdyz uz tak `changeSet.phoneNumber?.let { validationService.formatPhoneNumber(it) }`